content/docs/command-reference/add.md

```diff
@@ -428,3 +425,6 @@ system that can handle it), they can use `dvc pull` as usual:
 A       data.xml
 1 file added and 1 file fetched
 ```
+
+> For a similar operation that actually keeps a connection to the data source,
+> please see `dvc import-url`.
```

Comment:
👍 

I'll link it to https://dvc.org/doc/command-reference/import-url#example-transfer-to-remote-storage